docs/app/content-collections.md

```diff
@@ -1,168 +1,122 @@
 # Collection Page
 
-> This is the main gateway to your content items, providing a highly configurable Layout for browsing and visualizing
-> the items within a collection. The header of this page includes key action buttons for creating, deleting, and batch
-> editing items.
+> The Collection Page displays all Items within a Collection. The page comes with highly configurable Layouts for
+> browsing, visualizing, and managing these Items.
 
 [[toc]]
 
-## Creating an Item
+## Search Items
 
-1. Navigate to the collection where you want to add an item
-2. Click on the "Create Item" (+ icon) button located in the header
-3. Complete the Item Detail form
-4. Click the "Save Item" (✓ icon) button located in the header
+<video autoplay muted loop controls title="Search Items">
+	<source src="" type="video/mp4" />
+</video>
 
-::: tip Singletons
+Input a string or number into the search bar to find all Items that have one or more Fields that contain your string or
+number, not including any Items which have been [filtered out](#filter-items). To search through a Collection's Items,
+follow the steps below.
 
-If the Collection is configured as a "Singleton" in the data model settings, then the App will automatically skip step
-#2 above.
+1. Navigate to the Content Module and select the desired Collection.
+2. Select <span mi icon>search</span> in the Page Header and a search bar will appear.
+3. Type in the string or number to search by. The page will display Items that have a matching string or number value in
+   one or more of its Fields.
 
-:::
+## Filter Items
 
-## Manually Sorting Items
+![Filter Items](image.webp)
 
-1. Navigate to the [Content Module](/app/overview/#_1-module-bar)
-2. Navigate to the collection of the items you want to sort
-3. Choose a layout that supports manual sorting (eg: the default "table" mode)
-4. Enable manual sorting. (eg: for the table layout, click on the Sort column's header)
-5. Drag items by their handle into a new position
+It is possible to conditionally [filter](/app/filters/) the Items displayed on the Collection page. To filter a
+Collection's Items, follow the steps below.
 
-The new sort order is saved automatically, so make sure you wait until the App's loading indicator finishes before
-navigating away from the page.
+1. Navigate to the Content Module and select the desired Collection.
+2. Select <span mi icon>filter_list</span> in the Page Header.
+3. Apply any [Filters](/app/filters/) as needed.
 
-::: warning Requires Configuration
+## Manually Sort Items
 
-This feature is only available if a Sort column has been configured in the Collection's data model settings.
-
-:::
+<video autoplay muted loop controls title="Batch Edit Items">
+	<source src="" type="video/mp4" />
+</video>
 
-## Ordering Items
```

Comment:
Sort and Order: The naming convention is confusing. I want to call them both sorting or both ordering and denote with manual vs asc/desc.